src/algebra/group/commute.lean

```diff
@@ -115,6 +115,26 @@ theorem units_of_coe : commute (u₁ : M) u₂ → commute u₁ u₂ := semiconj
 @[simp, to_additive]
 theorem units_coe_iff : commute (u₁ : M) u₂ ↔ commute u₁ u₂ := semiconj_by.units_coe_iff
 
+@[to_additive] lemma is_unit_mul_iff {a b : M} (h : commute a b) :
```

Comment:
Should we reimplement `is_unit.mul_iff` in terms of this?

Reply:
This means moving it to `algebra/group/commute` or moving `commute` deeper in the `import` tree. Either option LGTM but I have no time for this.